test/integration/apis/redeem.test.ts

```diff
@@ -56,81 +64,84 @@ describe.skip("redeem", () => {
             assert.isRejected(redeemAPI.request(amount, randomDecodedAccountId));
         });
 
-        it("should page listed requests", async () => {
-            // requires PolkaBTC to have already been issued
-            const bob = keyring.addFromUri("//Bob");
-            redeemAPI.setAccount(alice);
-            const bobVaultId = api.createType("AccountId", bob.address);
-            const sentRequests = 4;
-            for (let i = 0; i < sentRequests; i++) {
-                const amount = api.createType("Balance", i);
-                await redeemAPI.request(amount, randomDecodedAccountId, bobVaultId);
-            }
+        it("should request and execute issue, request redeem", async () => {
+            const btcCore = new DefaultBTCCoreAPI("http://0.0.0.0:3002");
+            btcCore.initializeClientConnection("regtest", "0.0.0.0", "rpcuser", "rpcpassword", "18443", "Alice");
+            const blocksToMine = 3;
+            keyring = new Keyring({ type: "sr25519" });
+            alice = keyring.addFromUri("//Alice");
+            dave = keyring.addFromUri("//Dave");
 
-            const listingsPerPage = 2;
-            const requestsIterator = redeemAPI.getPagedIterator(listingsPerPage);
-            let curr = await requestsIterator.next();
-            let requestCount = 0;
-            while (!curr.done) {
-                requestCount += curr.value.length;
-                assert.isTrue(curr.value.length <= listingsPerPage);
-                curr = await requestsIterator.next();
-            }
-            assert.equal(requestCount, sentRequests);
-        });
-    });
+            // request issue
```

Comment:
Since you are reusing the same code below you can potentially move this to a separate function to avoid duplication.